chain/vm/runtime.go

```diff
@@ -163,7 +163,7 @@ func (rt *Runtime) ValidateImmediateCallerAcceptAny() {
 func (rt *Runtime) CurrentBalance() abi.TokenAmount {
 	b, err := rt.GetBalance(rt.Message().Receiver())
 	if err != nil {
-		rt.Abortf(err.RetCode(), "get current balance: %v", err)
+		vmabortf(err.RetCode(), "get current balance: %v", err)
```

Comment:
I might be misunderstanding this, but shouldn't this stay `rt.Abortf`?

If `rt.GetBalance` returns an error, it will be based on `aerrors.Escalate(err, "failed to look up actor balance")`, which is not setting `retCode`, so `retCode` would be 0...